sklearn/ensemble/_hist_gradient_boosting/gradient_boosting.py

```diff
@@ -1455,13 +1457,24 @@ class HistGradientBoostingClassifier(ClassifierMixin, BaseHistGradientBoosting):
 
     Parameters
     ----------
-    loss : {'auto', 'binary_crossentropy', 'categorical_crossentropy'}, \
-            default='auto'
-        The loss function to use in the boosting process. 'binary_crossentropy'
-        (also known as logistic loss) is used for binary classification and
-        generalizes to 'categorical_crossentropy' for multiclass
-        classification. 'auto' will automatically choose either loss depending
-        on the nature of the problem.
+    loss : {'log_loss'}, default='log_loss'
```

Comment:
I thought we agreed to keep all the valid values until end of deprecation cycle :D

Reply:
Right. Sorry. Short on time at the moment.

Reply:
Done